sdk/communication/AzureCommunicationChat/Source/Signaling/TrouterEventUtil.swift

```diff
@@ -104,28 +104,28 @@ public enum TrouterEventUtil {
     ///   - chatEventId: The ChatEventId, determines the type of ChatEvent that should be created.
     ///   - request: The request payload.
     /// - Returns: A chat event model.
-    public static func create(chatEvent chatEventId: ChatEventId, from request: TrouterRequest) throws -> Any {
+    public static func create(chatEvent chatEventId: ChatEventId, from _: TrouterRequest) -> EventType {
         switch chatEventId {
         case ChatEventId.chatMessageReceived:
-            return try ChatMessageReceivedEvent(from: request)
+            return EventType.chatMessageReceivedEvent
         case ChatEventId.typingIndicatorReceived:
-            return try TypingIndicatorReceivedEvent(from: request)
+            return EventType.typingIndicatorReceivedEvent
```

Comment:
This logic doesn't do anything. It just changes the ChatEventId enum value into an identical EventType enum value.